3.9/graphs-smart-graphs.md

```diff
@@ -71,6 +71,17 @@ overhead. The more subgraphs are touched the more network cost will apply.
 However the overall performance is never worse than the same query using a
 General Graph.
 
+Benefits of Hybrid SmartGraphs
+-------------------------------
+
+Hybrid SmartGraphs are capable of using Satellite Collections within their graph
+definition. Therefore, defined EdgeDefinitions between SmartCollections and
+SatelliteCollections can be created. As Satellite Collections are globally
+replicated to each participating DB-Server, graph traversals, shortest path,
+weighted traversals and k-shortest-paths queries can be executed partially
+locally on each DB-Server. This means a larger part of the query can be executed
+fully local whenever data from the Satellite part is required.
+
```

Comment:
A note that edge collections between smart- and satellite-collections are replicated like satellite collections (if I got that correctly) seems appropriate, as the amount of space needed could be unexpected otherwise.